### ISSUE TITLE
Python: wrong result for simple linear programing problem

### ISSUE THREAD

**Issue by Irvingouj** [google/or-tools#3721]
**What version of OR-Tools and what language are you using?**
Version: 
pip list gives
ortools  9.5.2237
language:
python
**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
GLOP
**What operating system (Linux, Windows, ...) and version?**
Windows

**What did you do?**
```py
from ortools.linear_solver import pywraplp

solver = pywraplp.Solver.CreateSolver('GLOP')
param = pywraplp.MPSolverParameters()
param.SetIntegerParam(pywraplp.MPSolverParameters.PRESOLVE, pywraplp.MPSolverParameters.PRESOLVE_OFF)
x = solver.NumVar(-solver.infinity(), solver.infinity(), 'x')
y = solver.NumVar(-solver.infinity(), solver.infinity(), 'y')
solver.Add(-4.29*x + 5.14*y <= 828.8)
# 1.36x + 8.76y <= 271.2
solver.Add(1.36*x + 8.76*y <= 271.2)
# -4.45x + -2.4y <= -430.23
solver.Add(-4.45*x + -2.4*y <= -430.23)
# 0.65x + 4.37y <= -281.35
solver.Add(0.65*x + 4.37*y <= -281.35)
# max 8.77x + -0.09y
solver.Maximize(8.77*x-1*y)

status = solver.Solve(param)

print(status == pywraplp.Solver.INFEASIBLE)
```

**What did you expect to see**
should print false as it is feasible
**What did you see instead?**
print(status == pywraplp.Solver.INFEASIBLE) give true

the point (700,-400) is feasible, see the attaching image as a reference
![ortool_glop_bug](https://user-images.githubusercontent.com/49964599/228394347-c0a0cb76-0a1f-4d15-9e71-4489367a6e11.PNG)

**Anything else we should know about your project / environment**

python 3.10.10

**Comment by lperron** [comment]
Can you turn or the log ?

Can you try with clp?

**Comment by Irvingouj** [comment]
Thanks, it is resolved in 9.6, help appreciated!
